examples/swapAndBridge.ts

```diff
@@ -71,6 +81,47 @@ const swapAndBridge: Parameters<typeof withAnvilProvider>[0] = async (
     buyTokenBalance: OrderBalance.ERC20,
   };
 
+  // weiroll required since swap output amount is not known
+  // at tx time. need to fetch it at execution time.
+  const weirollCommands = [
+    // balanceOf staticcall
+    encodeCommand(
+      fnSelector('balanceOf(address)'),
+      encodeFlag(false, false, CallType.StaticCall),
+      encodeInput(
+        encodeInputArg(true, 0),
+        END_OF_ARGS,
+        END_OF_ARGS,
+        END_OF_ARGS,
+        END_OF_ARGS,
+        END_OF_ARGS
+      ),
+      encodeInputArg(true, 1),
+      USDC
+    ),
+    // bridge relay tokens call
+    encodeCommand(
+      fnSelector('relayTokens(address,address,uint256)'),
+      encodeFlag(false, false, CallType.Call),
+      encodeInput(
+        encodeInputArg(true, 2), // token
+        encodeInputArg(true, 0), // user address
+        encodeInputArg(true, 1), // balance
+        END_OF_ARGS,
+        END_OF_ARGS,
+        END_OF_ARGS
+      ),
+      END_OF_ARGS,
+      GNOSIS_CHAIN_BRIDGE
+    ),
+  ];
+
+  const state = [
+    hexZeroPad(proxyAddress, 32), // address to query the balance of
+    '0x', // this is where balance output will be written
+    hexZeroPad(USDC, 32), // USDC token address
+  ];
```

Comment:
Explain in more detail what is happening here. It's not clear why the `balanceOf` is in the previous weiroll commands, but then there is state here.